form/unit_testing.rst

```diff
@@ -211,6 +221,21 @@ allows you to return a list of extensions to register::
         // ... your tests
     }
 
+.. note::
+
+    By Default only the
+    :class:`Symfony\\Component\\Form\\Extension\\Core\\CoreExtension` is
+    registered in tests. You can find other extensions from the Form component
+    in the ``Symfony\Component\Form\Extension`` namespace.
+
+.. caution::
+
+    When your type relies on the ``EntityType``, you should register the
```

Comment:
I feel like this is in a bit weird place. If you made it this far in the article, it seems like you already would know that you needed an extra form extension. I would rather move it up in the article (and then linking to this section, like you already did in another spot).

Also, I would vote to not describe `KernelTestCase` here, as this article is about Unit testing and using a booted kernel is no longer unit testing.

Reply:
>I feel like this is in a bit weird place. If you made it this far in the article, it seems like you already would know that you needed an extra form extension. I would rather move it up in the article (and then linking to this section, like you already did in another spot).

You're, right I was not really happy with this place too.

>Also, I would vote to not describe KernelTestCase here, as this article is about Unit testing and using a booted kernel is no longer unit testing.

I don't agree on this one, I don't think we're showing a unit test here (as it would apply to test e.g the `buildForm` call only with `$type = new SomeType(); $type->buildForm()`.
I think this is more an integration test with the Form component using its provided base integration test case. When this is not enough, there is the `KernelTestCase` to use the form factory built by the container for the kernel integration, which is needed to solve the related issue in core.

In result I've moved the block and reword a few things, let me know what you think, thanks!